libraries/Microsoft.Bot.Expressions.Properties/Extensions.cs

```diff
@@ -35,12 +36,14 @@ public static object Evaluate(this LGFile lgFile, string inlineStr, object scope
             inlineStr = !inlineStr.Trim().StartsWith(multiLineMark) && inlineStr.Contains('\n')
                    ? $"{multiLineMark}{inlineStr}{multiLineMark}" : inlineStr;
 
-            var wrappedStr = $"# {fakeTemplateId} \r\n - {inlineStr}";
-
-            var newContent = $"{lgFile.Content}\r\n{wrappedStr}";
+            var newContent = $"# {fakeTemplateId} \r\n - {inlineStr}";
 
             var newLgFile = LGParser.ParseText(newContent, lgFile.Id, lgFile.ImportResolver);
-            return newLgFile.EvaluateTemplate(fakeTemplateId, scope);
+
+            var memory = SimpleObjectMemory.Wrap(scope);
```

Comment:
move this line at the end, or perhaps remove this, if CustomizedMemory already try to wrap memory inside.